src/connectors/notify/notify.connector.js

```diff
@@ -63,6 +75,14 @@ const sendSingleEmail = async (
 
     const notifyResponse = await notifyClient.sendEmail(...notifyArguments);
     const responseBody = notifyResponse.body;
+    if (updateCache) {
```

Comment:
I don't think passing updateCache as an argument is a good idea. You're changing signature of a function that's used elsewhere where it doesn't need to supply this object. Since we're awaiting result of sendSingleEmail, we can simply update cache after that call

Reply:
I agree, just couldn't think of a way to handle the failure outside of this function. Due to other changes to change the status for notifications from a success / failure to a sent. This has been removed